routes/web.php

```diff
@@ -139,6 +139,11 @@
     Route::post('blogs/moderate/{id}', 'Blogs\BlogsController@moderate')->name('blogs.moderate');
     // end - blogs
 
+    // START - shop
+    Route::get('shop/products/bulk-upload', 'Shop\ProductsController@showBulkUpload')->name('shop.show-bulk-upload');
+    Route::post('shop/products/bulk-upload', 'Shop\ProductsController@bulkUpload')->name('shop.bulk-upload');
```

Comment:
Enhancements (can be in the future):
- can use new syntax for this, please check master `Route::get('...', [ProductsController::class, 'showBulkUpload'])` L8 compatibility if we remove namespaces on RouteServiceProvider
- use `shop.products` instead to future proof  eg.`->name('shop.products.show-bulk-upload')`